optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/persistence/VehicleRoutingImporter.java

```diff
@@ -391,6 +391,12 @@ private void createVehicleList() {
                 vehicle.setId(id);
                 id++;
                 vehicle.setCapacity(capacity);
+                //Multiple Depot VRP assign one vehicle for each depot
+                if(depotList.size() == vehicleListSize){
+                    vehicle.setDepot(depotList.get(i));
```

Comment:
There might be more depots than vehicles or there might be less.
Probably something like depotList.get(i % depotList.size()) should be used.